### DISCUSSION TITLE
Revert the change on secret of a repo

### DISCUSSION THREAD

**Discussion by finorix079** [community/community#29727] [Other Feature Feedback, Questions, & Ideas]
Hi everyone, I am new to here. I just wonder if there's a possible way to revert the change I have made on my secret. I made a change and my github action no longer work so I want to go back and see what is happening.

**Comment by airtower-luna** [comment]
There's no undo, you'd have to edit the secret again and put in the first value again, if you still have it yourself. If you don't you'll have to try with new values for whatever you're doing.
